crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
+
+The extension allows you to test this service for free, even if you don't connect the extension to a
+database. After a while, the extension asks you to buy credits to keep using this additional service.
+
+.. image:: media/outlook-extension-credits-message.png
+   :align: center
+   :alt: "Could not auto-complete the company: not enough credits!" warning message in the Outlook
+         extension
+
+You can disregard this message and keep using the extension for free without *Lead Enrichment* or
+buy more *Lead Generation* credits.
```

Comment:
I would write this sentence like this: "You can disregard this message, keep using the extension for free without *_Lead Enrichment_*, or buy more _*Lead Generation*_ credits.

Reply:
To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

Reply:
> I would write this sentence like this: "You can disregard this message, keep using the extension for free without __Lead Enrichment__, or buy more __Lead Generation__ credits.

Thanks, but it is not three different possibilities, but only two:
all that is before "or", and what's after.
Would a comma before "or" make it more understandable?